alcs-frontend/src/app/shared/card/card.component.ts

```diff
@@ -28,5 +31,23 @@ export class CardComponent implements OnInit {
     this.cardSelected.emit(cardId);
   }
 
-  ngOnInit(): void {}
+  private getLatestDecisionDate() {
+    return new Date(
+      Math.max(
+        ...this.cardData.decisionMeetings.map((element) => {
+          return new Date(element.date).valueOf();
```

Comment:
This looks like a redundant conversion, looks like its going Number -> Date -> Number -> Date. I think you can remove the new Date() from inside the map.

Reply:
nope, without conversion it return Invalid Date

Reply:
My bad, element.date is a string